src/Robo/Task/Discovery/Tasks.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Usher\Robo\Task\Discovery;
+
+trait Tasks
+{
+    /**
+     * @param string $command
+     * @param array $alternatives
+     *
+     * @return \Usher\Robo\Task\Discovery\Alternatives|\Robo\Collection\CollectionBuilder
+     */
+    protected function taskAlternatives(string $command, array $alternatives)
```

Comment:
I'm curious why you aren't using this trait in the `DevelopmentModeBaseCommands` class.

Reply:
🙈

Reply:
> 🙈

Super confusing. To add a task, you need to define a trait that is then used in the Robo file. The trait loads your task. It's a trip.

Reply:
Ok, that makes me feel better, as I was struggling to piece this all together.